### ISSUE TITLE
Write a short and descriptive title!

### ISSUE THREAD

**Issue by Myriad-Dreamin** [typst/typst#2434]
### Description

Compiling [arch-media-box-typst](https://github.com/Myriad-Dreamin/arch-media-box-typst) and get following panic on **latest** typst:

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', crates\typst\src\export\pdf\page.rs:472:77
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

The same document doesn't trigger this crash at exact typst v0.8.0.

### Operating system

Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by Dherse** [comment]
Oh that's very bad, you have non inversible transforms 😬

**Comment by Dherse** [comment]
In SVG & PNG the transforms are invertible which means that it's likely due to an issue in PDF export. I am investigating this and I'll try to get a patch out today.

**Comment by Dherse** [comment]
Indeed, the issue was caused by missing a `.abs()` in the inverse transform computation: it would check whether the determinant of the transform is smaller than some tolerance (1e-12 in this case), but if the determinant is negative (which is possible given some rotations) it would always panic. This will be fixed in the PR I am opening up.
